src/phase0/state_transition/mod.rs

```diff
@@ -761,20 +761,21 @@ pub fn get_beacon_committee<
     slot: Slot,
     index: CommitteeIndex,
     context: &Context,
-) -> Result<&[ValidatorIndex], Error> {
+) -> Result<Vec<ValidatorIndex>, Error> {
     let epoch = compute_epoch_at_slot(slot, context);
-    let committees_per_slot = get_committee_count_per_slot(state, epoch);
+    let committees_per_slot = get_committee_count_per_slot(&state, epoch, context);
```

Comment:
we will want to pass the state by reference most places

so the type signature can change to `&BeaconState` in the fn defn

and we don't need to keep taking `&` inside the fn body

Reply:
fixed